src/disease.c

```diff
@@ -185,12 +186,26 @@ void transmit_virus_by_type(
 			{
 				interaction = infector->interactions[ model->interaction_day_idx ];
 
+                //Determine the effect hospitalisation has on the hazard rate.
+                if( model->params->hospital_on )
+                {
+                    switch( infector->hospital_state )
+                    {
+//                        case WAITING:       hospital_state_modifier = model->params->waiting_infectivity_modifier; break;
```

Comment:
Are al these comments because we're waiting of future implementation?

Reply:
Uncommented for now - not sure why it was commented out in the first place. Will check with the others whether we were debugging this or not, but to my knowledge, this should be working.